definitions/divorce/json/AuthorisationComplexType/AuthorisationComplexType-roles-and-permissions-resp-journey-nonprod.json

```diff
@@ -0,0 +1,34 @@
+[
+  {
+    "LiveFrom": "26/01/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "RespondentOrganisationPolicy",
+    "ListElementCode": "OrgPolicyCaseAssignedRole",
+    "UserRole": "[RESPSOLICITOR]",
```

Comment:
it seems only the "caseworker-caa" needs to have permissions for fields inside the complex type :)
So you would need to replace [RESPSOLICITOR] with caseworker-caa
(see https://github.com/hmcts/finrem-ccd-definitions/blob/230139d4c0fe134cdf216821ed5e7cae054d0d9d/definitions/contested/json/AuthorisationComplexType/AuthorisationComplexType-respSol-nonprod.json#L6)

Reply:
ive updated this area to better reflect the Updated roles, in this commit - a0d346f7391292391a83f5bd635638ae9538f753